### ISSUE TITLE
readme false advertising

### ISSUE THREAD

**Issue by fersegundo2** [kaisellgren/mailer#238]
The readme leads to believe that it is possible to send an email from flutter using code by JohannesMilke. This is from my point of view not true or at least very inaccurate. What most users would want to do and would assume by the readme that is possible is:
use case 1- send an email from the email of the app user to any other user
use case 2 - send an email from the developers email to any other email

use case 2 is not possible 
use case 1 is possible ONLY if we add the users email as "test user" in google cloud OAuth consent screen. Which obviously is not what we want because: 1) we want this feature to work in production mode 2) we cannot add all future users as "test users"

So from what I understand this library can be used from flutter only for testing/developer logging. Please specify this in the readme

**Comment by close2** [comment]
@fersegundo2 
I haven't tested the tutorial, but I thought that use case 2 is possible:

use case 2 should be possible: https://youtu.be/RDwst9icjAY?t=834

The test users should only be necessary in testing mode.

Am I mistaken?

**Comment by bsutton** [comment]
I don't think this is a case of false advertising just a matter of assumptions about the reader's  knowledge.

The doco clearly talks about SMTP which to an experienced dev makes it obvious what the package is about.

Having said that, some explicit wording around the fact that it's an SMTP client will help out newbies.

**Comment by close2** [comment]
I have improved the documentation.
